lib/server-code/runners/tasks/executor.js

```diff
@@ -111,6 +111,10 @@ function enrichTask(task, opts) {
     task.___jsonclass = executor.RAI
     task.actionType = SHUTDOWN_ACTION
   }
+
+  task.invocationContextDto.response = {
+    headers: task.invocationContextDto.httpResponseHeaders || {},
```

Comment:
should be an array, not an object

Reply:
why should it be an array?, in Java it's a Map
https://github.com/Backendless/Android-SDK/blob/master/src/com/backendless/servercode/InvocationContext.java#L47